core/src/main/se/tevej/game/view/gui/InventoryElement.java

```diff
@@ -14,12 +17,34 @@
     public InventoryElement(GuiFactory guiFactory, int amount,
                             String imagePath, ResourceType resourceType) {
         this.resourceType = resourceType;
-        label = guiFactory.createLabel().text(Integer.toString(amount));
+        label = guiFactory.createLabel().text(parseDoubleWithSuffix(amount));
         image = guiFactory.createImage().image(imagePath);
     }
 
     public void update(int newAmount) {
-        label.text(Integer.toString(newAmount));
+        label.text(parseDoubleWithSuffix(newAmount));
+    }
+
+    private String parseDoubleWithSuffix(double amount) {
+        double result;
+        int[] suffixNumbers = {1000, 1000000};
+        String suffix = "";
+        if (amount < suffixNumbers[0]) {
+            result = amount;
+        } else if (amount < suffixNumbers[1]) {
+            result = amount / 1000;
```

Comment:
Why do you not use suffixNumbers[1] here?

Reply:
oh, lol
det var tanken. glömde bara heheh. Tack för upplysningen